gTea/app/src/main/java/com/nechaieva/gtea/utils/LevenshteinOrderProcessor.java

```diff
@@ -58,7 +61,17 @@ public void loadMenu(ArrayList<String> menu) {
      * A menu item.
      */
     public Optional<String> findInMenu(String query, int sensitivity) {
+
+        if (menu == null) {
+            throw new NullPointerException("Menu is not initialized");
```

Comment:
Did you consider alternative exception types?

Did you consider alternative designs which avoid this failure case at all?